docs/guide/running-extern-robot-controllers.md

```diff
@@ -44,6 +44,13 @@ Webots environment variables needed by extern controllers:
 
 &nbsp;
 
+**Python**: Add the correct python version:
+
+When setting or extending the `PYTHONPATH` environment variable, make sure to replace the `X` in `python3X` with your python version.
+This can be found by typing `python3 --version` in the command line. If for example the answer is `Python 3.8.10`, `python3X` should be `python38`.
```

Comment:
On Windows, the "python3" command is not found from the cmd.exe, instead "python" should be used.
Also we usually write one sentence per line.
```suggestion
This can be found by typing `python --version` in the command line.
If for example the answer is `Python 3.8.10`, `python3X` should be `python38`.
```